src/http/client.rs

```diff
@@ -1430,6 +1431,14 @@ impl Http {
                     multipart = multipart
                         .file(file_num.to_string(), path)?;
                 },
+                AttachmentType::Image(url) => {
+                    let mut picture: Vec<u8> = vec![];
+                    let req = &mut self.client.get(url).send()?;
+                    let _ = std::io::copy(&mut *req, &mut picture);
+                    multipart = multipart
+                        .part(file_num.to_string(), Part::bytes(Cow::Borrowed(&picture[..]).into_owned())
+                            .file_name(format!("{}.webp", file_num)));
```

Comment:
This should use the actual file extension unless you are converting the image to webp format.